app/views/admin/suppliers/index.html.haml

```diff
@@ -0,0 +1,17 @@
+.govuk-grid-row
+  .govuk-grid-column-full
+    %h1.govuk-heading-xl
+      Suppliers
+
+.govuk-grid-row
+  .govuk-grid-column-full
+    %table.govuk-table{:class => 'govuk-!-margin-top-7'}
+      %thead.govuk-table__head
+        %tr.govuk-table__row
+          %th.govuk-table__header Name
+      %tbody.govuk-table__body
+        - @suppliers.each do |supplier|
```

Comment:
Would we want to cover the case when there are no suppliers, and avoid printing an empty table, or is that such a remote possibility that it's not worth coding for?

Reply:
It's definitely remote, but now I noticed that a search that returns no results has no empty state. I will try and fix that and the suppliers = 0 case.